content/tor-browser-for-android/content.lr

```diff
@@ -0,0 +1,67 @@
+_model: topic
+---
+title: Tor Browser for Android
+---
+description: Learn about Tor Browser for Android and how to install it
+---
+seo_slug: tor-broser-for-android
+---
+body:
+
+##### INTRODUCTION - A LITTLE BIT OF HISTORY
+
+[Mobile browsing](https://www.theguardian.com/technology/2016/nov/02/mobile-web-browsing-desktop-smartphones-tablets) has increasingly become the primary way many people go online.
+In 2014, Tor Browser was used by millions of people on their desktop computers, but there was not yet a way to use Tor on a mobile device.
+We recognized the need for protecting people's rights to privacy and freedom on mobile devices, however, as a small nonprofit organization, we were not yet ready to meet that demand.
+That's when our friends at Guardian Project began exploring the possibility of providing Tor Browser on Android themselves. 
+Their answer was [Orfox](https://guardianproject.info/apps/orfox/).
+
+
+Orfox was built from the same source code as Tor Browser (which is built upon Firefox), with a few minor modifications to the privacy enhancing features to make them compatible with Firefox for Android and the Android operating system.
```

Comment:
It's nice to have an introduction about Orfox, Orbot and the new Tor Browser for Android, but I don't see why we should maintain a whole history section about Tor development on mobile devices in a User Manual.